project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "cowswap",
+  "$schema": "node_modules/nx/schemas/project-schema.json",
+  "targets": {
+    "local-registry": {
+      "executor": "@nx/js:verdaccio",
+      "options": {
+        "port": 4873,
+        "config": ".verdaccio/config.yml",
+        "storage": "tmp/local-registry/storage"
+      }
+    }
+  }
```

Comment:
Why did you add this file? 

It sounds good to have our own local registry, but not sure should have been part of this PR. How would like us to test it?

Reply:
Automatically created.
Will remove verdaccio and all of this.